# the desired pose of the tool frame
geometry_msgs/PoseStamped pose

# tool frame defauts to x_wrist_roll_link
geometry_msgs/Pose tool_frame

# the joint space configuration to resolve redundancy
sensor_msgs/JointState ik_seed

# Weight matrices for IK solver
kdl_arm_kinematics/KDLWeights weights

---
sensor_msgs/JointState solution

motion_planning_msgs/ArmNavigationErrorCodes error_code
